app/containers/Network/containers/BulkEditAccessPoints/index.js

```diff
@@ -0,0 +1,330 @@
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'MIN LOAD',
+    dataIndex: 'minLoad',
+    key: 'minLoad',
+    editable: true,
+    render: renderTableCell,
+  },
+];
+
+const BulkEditAPs = () => {
+  const { customerId } = useContext(UserContext);
+  const [selectedLocationIds, setSelectedLocationIds] = useState([]);
+  const { id } = useParams();
+  const { data } = useQuery(GET_ALL_LOCATIONS, {
+    variables: { customerId },
+  });
+  const [
+    filterEquipment,
+    {
+      loading: filterEquipmentLoading,
+      error: filterEquipmentError,
+      refetch,
+      data: equipData,
+      fetchMore,
+    },
+  ] = useLazyQuery(FILTER_EQUIPMENT_BULK_EDIT_APS, {
+    variables: { customerId },
+  });
+  const [updateEquipmentBulk] = useMutation(UPDATE_EQUIPMENT_BULK);
+
+  const fetchFilterEquipmentBulkEditAps = locationIds => {
+    filterEquipment({
+      variables: {
+        customerId,
+        locationIds,
+        equipmentType: 'AP',
+      },
+    });
+  };
+
+  const locationIdsArray = [];
+  const getChildNodes = locations => {
+    const childArr = [];
+    locations.forEach(a => {
+      locationIdsArray.push(a);
+      data.getAllLocations.forEach(b => {
+        if (b.parentId === a) childArr.push(b.id);
+      });
+    });
+    if (childArr.length > 0) {
+      getChildNodes(childArr);
+    } else {
+      setSelectedLocationIds(locationIdsArray);
+    }
+  };
+
+  useEffect(() => {
+    const locationId = parseInt(id, 10);
+    getChildNodes([locationId]);
+  }, [data, id]);
+
+  useEffect(() => {
+    fetchFilterEquipmentBulkEditAps(selectedLocationIds);
+  }, [selectedLocationIds]);
+
+  const getRadioDetailsByType = (radioDetails, type, radioType) => {
+    if (type === 'cellSize') {
+      const cellSize = radioDetails.radioMap[radioType].rxCellSizeDb.value;
+      return cellSize;
+    }
+    if (type === 'probeResponseThreshold') {
+      const probeResponseThreshold =
+        radioDetails.radioMap[radioType].probeResponseThresholdDb.value;
+      return probeResponseThreshold;
+    }
+    if (type === 'clientDisconnectThreshold') {
+      const clientDisconnectThreshold =
+        radioDetails.radioMap[radioType].clientDisconnectThresholdDb.value;
+      return clientDisconnectThreshold;
+    }
+    if (type === 'snrDrop') {
+      const snrDrop = radioDetails.advancedRadioMap[radioType].bestApSettings.dropInSnrPercentage;
+      return snrDrop;
+    }
+    // if (type === 'minLoad') {
+    const minLoad = radioDetails.advancedRadioMap[radioType].bestApSettings.minLoadFactor;
+    return minLoad;
+    // }
+  };
+
+  const getRadioDetails = (radioDetails, type) => {
+    const is5GHzU = getRadioDetailsByType(radioDetails, type, IS_5GHZU);
+    const is5GHzL = getRadioDetailsByType(radioDetails, type, IS_5GHZL);
+    const is2dot4GHz = getRadioDetailsByType(radioDetails, type, IS_2DOT4GHZ);
+
+    return {
+      is5GHzU,
+      is5GHzL,
+      is2dot4GHz,
+    };
+  };
```

Comment:
the radios can be any combination per AP, instead loop through advancedRadioMap and radioMap like 
`Object.keys(radioDetails.advancedRadioMap).map(i => radioDetails.advancedRadioMap[i][dataIndex])`